src/main.rs

```diff
@@ -286,3 +301,10 @@ fn on_notify_ev(
 		}
 	}
 }
+fn parse_color_to_i32(cs: &str) -> Result<i32, csscolorparser::ParseColorError> {
+	let color = csscolorparser::parse(cs)?;
+	let [r, g, b, _] = color.to_rgba8();
+	let u: u32 = u32::from(r) * 256 * 256 + u32::from(g) * 256 + u32::from(b);
+	let bytes = u.to_le_bytes();
+	Ok(i32::from_le_bytes(bytes))
```

Comment:
```suggestion
	Ok(i32::from_be_bytes([0, r, g, b]))
```
This is equivalent, as far as I can tell.